### PR TITLE
Allow changing the path to initdb

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: tasks/configure.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
 # Using sudo because become_user may be overridden
 - name: Initialize database
-  command: sudo -Hu {{ postgres_user }} initdb --locale {{ postgres_locale }} -E {{ postgres_encoding }} -D {{ postgres_data_directory }}
+  command: sudo -Hu {{ postgres_user }} {{ postgres_initdb | default('initdb') }} --locale {{ postgres_locale }} -E {{ postgres_encoding }} -D {{ postgres_data_directory }}
```
Comment (SlothOfAnarchy) [inline tasks/configure.yml]: why not use the `defaults` folder?

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: tasks/configure.yml (hunk 1 above)
Comment (dasJ) [reply tasks/configure.yml]: Why should I?

Comment (SlothOfAnarchy) [reply tasks/configure.yml]: Because all default values would then reside in the same place. You even recommend to look vars up there in the readme ;)

**Comment by dasJ** [comment]
@SlothOfAnarchy Pls rereview

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**